web_environment_ribbon/static/src/css/ribbon.css

```diff
@@ -14,7 +14,7 @@
     -moz-transform: rotate(-45deg);
     -o-transform: rotate(-45deg);
     transform: rotate(-45deg);
-    z-index: 1000;
+    z-index: 9999;
```

Comment:
This was intended, because long texts are partially hidden behind the menu instead.

If the ribbon can completely hide small menus, maybe making it thicker (10 px less) can avoid the issue ?